### DISCUSSION TITLE
ABILITA RECUPERO PASSWORD

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#291] [Richieste di aiuto]
Buongiorno ,come si fa ad abilitare un utente a recuperare la propria password all'email?

**Comment by sempliciniluigi89** [comment]
Buongiorno ,come si fa ad abilitare un utente a recuperare la propria password all'email? Lo chiedo perché dal lato segreteria abbiamo registrato tutti gli indirizzi mail dei docenti ,i quali hanno provato a recuperare la loro password inserendo l'indirizzo mail registrato presso la segreteria ed appare un simile messaggio :l'utente non autorizzato al recupero della password

**Comment by trinko** [comment]
Si, questo è il comportamento attuale per i docenti, perché, per rafforzare la sicurezza, è richiesto il codice OTP.

Devi attivare il codice OTP dal profilo docente.
